<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>stu_money</class>
 <widget class="QDialog" name="stu_money">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>606</width>
    <height>379</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>12</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>90</y>
     <width>431</width>
     <height>101</height>
    </rect>
   </property>
   <property name="title">
    <string/>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="2" column="0">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>起止日期</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QDateEdit" name="begin_time">
      <property name="currentSection">
       <enum>QDateTimeEdit::YearSection</enum>
      </property>
      <property name="displayFormat">
       <string>yyyy/MM/dd</string>
      </property>
      <property name="calendarPopup">
       <bool>true</bool>
      </property>
      <property name="date">
       <date>
        <year>2021</year>
        <month>9</month>
        <day>1</day>
       </date>
      </property>
     </widget>
    </item>
    <item row="2" column="2">
     <widget class="QDateEdit" name="end_time">
      <property name="currentSection">
       <enum>QDateTimeEdit::YearSection</enum>
      </property>
      <property name="displayFormat">
       <string>yyyy/MM/dd</string>
      </property>
      <property name="calendarPopup">
       <bool>true</bool>
      </property>
      <property name="date">
       <date>
        <year>2021</year>
        <month>9</month>
        <day>1</day>
       </date>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>学号</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1" colspan="2">
     <widget class="QPlainTextEdit" name="stu_ID"/>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="res">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>210</y>
     <width>431</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>xx同学共消费x元</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>260</y>
     <width>92</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>查询</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>310</y>
     <width>92</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>退出</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton_2</sender>
   <signal>clicked()</signal>
   <receiver>stu_money</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>485</x>
     <y>324</y>
    </hint>
    <hint type="destinationlabel">
     <x>302</x>
     <y>189</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
